lava-v2-jobs-from-api.py

```diff
@@ -96,10 +124,9 @@ def main(args):
                         exit(1)
             if build['kernel_image']:
                 # handle devices without a DTB, hacky :/
+                build['dtb_dir_data'].append('qemu')
```

Comment:
Re: QEMU boots in every lab.  Personally, I don't want my lab bogged down with QEMU boots, so I won't setup QEMU devices.

I think we should use lava-docker to setup a lab in the cloud that does all the QEMU boots so that real labs can focus on real hardware.